addons/sale_stock/tests/test_sale_stock.py

```diff
@@ -1441,8 +1441,8 @@ def test_backorder_and_decrease_sol_qty(self):
             {'location_id': stock_location.id, 'location_dest_id': out_location.id, 'product_uom_qty': 1.0, 'quantity_done': 0.0, 'state': 'assigned'},
             {'location_id': out_location.id, 'location_dest_id': customer_location.id, 'product_uom_qty': 1.0, 'quantity_done': 0.0, 'state': 'waiting'},
         ])
-        self.assertEqual(ship01.move_ids.move_orig_ids, (pick01 | pick02).move_ids)
-        self.assertEqual(ship02.move_ids.move_orig_ids, (pick01 | pick02).move_ids)
+        self.assertEqual(ship01.move_ids.move_orig_ids, pick01.move_ids)
+        self.assertEqual(ship02.move_ids.move_orig_ids, pick02.move_ids)
```

Comment:
You can squash the 3 commits, the tests and the code are strongly working together